### ISSUE TITLE
conceal highlight issue for latex

### ISSUE THREAD

**Issue by xarthurx** [cocopon/iceberg.vim#69]
Environment:
- Windows
- Neovim  latest

If you have a trouble with coloration and are using Vim/Neovim with terminal:
- [x] Does your Vim support full colors? (`:set termguicolors`)
- [x] Does your terminal support full colors? (e.g. Terminal.app doesn't support it)

I'd like use the settings:

hi Conceal ctermfg=NONE
hi Conceal ctermbg=NONE
hi Conceal guifg=NONE
hi Conceal guibg=NONE

to make concealed chars and words similar to the background.
And it seems there's a bug for this fantastic iceberg theme:

iceberg:
![image](https://user-images.githubusercontent.com/1921878/86405523-cf37df00-bcb1-11ea-94e5-81edc21c9b49.png)

OceanicNext:
image
![image](https://user-images.githubusercontent.com/1921878/86405548-da8b0a80-bcb1-11ea-9958-c47058530b31.png)

And I found some post here:
kaicataldo/material.vim#12
for the 'material` theme, as they add some defined colour to the scheme.

Could we also do similar things for Iceberg?

**Comment by cocopon** [comment]
Could you share the example code about the problem? I'll check it in my environment later.

**Comment by xarthurx** [comment]
> Could you share the example code about the problem? I'll check it in my environment later.

You can simply try this:

```
\begin{equation} \nonumber
\min_{s(\bx)} \, \, \,  \int_\Omega \| \nabla s(\mathbf{x}) - \mathbf{g}(\mathbf{x}) \|^2 d\bx \quad \text{s.t.} \quad s(\mathbf{x}) = \begin{cases}
0 \, \, \,  \text{for} \, \, \, \bx \in \partial \Omega_0\\
1 \, \, \, \text{for} \, \, \, \bx \in \partial \Omega_1.\\
\end{cases}
\end{equation}

```


BTW, I found a solution that adding this:
```
au Colorscheme * hi! link Conceal Number
```
to my $VIMRC will resolve the issue.

But there for sure is some difference between your theme and the `oceanic` one if you'd like to explore.

**Comment by cocopon** [comment]
Thank you for your help! Fixed in #72.
![image](https://user-images.githubusercontent.com/602961/87240270-07c67e00-c453-11ea-8c4c-9d06315eae3b.png)
